apps/src/templates/sectionAssessments/MultipleChoiceAnswerCell.jsx

```diff
@@ -24,10 +24,26 @@ class MultipleChoiceAnswerCell extends Component {
   static propTypes = {
     percentValue: PropTypes.number.isRequired,
     isCorrectAnswer: PropTypes.bool,
+    textAnswered: PropTypes.string,
```

Comment:
this is the selected answer choice?

Reply:
Yes, it is.  I changed "textAnswered" to "displayAnswer" for clarity.